dotnet-desktop-guide/net/wpf/events/snippets/how-to-create-a-custom-routed-event/vb/CodeSampleVb.vbproj

```
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net6.0-windows</TargetFramework>
+    <RootNamespace>CodeSampleVb</RootNamespace>
+    <NeutralLanguage>en-us</NeutralLanguage>
+    <UseWPF>true</UseWPF>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Import Include="System.Windows" />
+    <Import Include="System.Windows.Controls" />
+    <Import Include="System.Windows.Data" />
+    <Import Include="System.Windows.Documents" />
+    <Import Include="System.Windows.Input" />
+    <Import Include="System.Windows.Media" />
+    <Import Include="System.Windows.Media.Imaging" />
+    <Import Include="System.Windows.Navigation" />
+    <Import Include="System.Windows.Shapes" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="WpfControlLibraryVb">
+      <HintPath>WpfControlLibraryVb.dll</HintPath>
+    </Reference>
+  </ItemGroup>
```

See my previous comment
```suggestion
  <ItemGroup>
    <ProjectReference Include="..\WpfControlLibraryCsharp\WpfControlLibrary.csproj" />
  </ItemGroup>
```